packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -103,19 +103,32 @@ class OTelReporter {
     userContext.vars['__otlStartTime'] = startTime;
     const span = trace
       .getTracer('artillery-tracer')
-      .startSpan('http_request', { startTime });
-    span.setAttribute('kind', 'client');
+      .startSpan(req.method.toLowerCase(), {
```

Comment:
The linked doc seems to say that the span name should be:
 
`{http.request.method} {http.route}`

i.e. we should use the URL or the request name if available.

Reply:
@hassy The `{http.request.method} {http.route}` is only applied in the case of HTTP server spans. It says:

> HTTP client spans have no `http.route` attribute since client-side instrumentation is not generally aware of the "route", and therefore HTTP client spans SHOULD use `{http.method}`

Reply:
The suggestion I made about implementing a `useRequestName` setting came from the next sentence in the same doc:

> Instrumentation MUST NOT default to using URI path as span name, but MAY provide hooks to allow custom logic to override the default span name.